packages/wxt/src/core/builders/vite/plugins/noopBackground.ts

```diff
@@ -10,13 +10,21 @@ export function noopBackground(): Plugin {
   const resolvedVirtualModuleId = '\0' + virtualModuleId;
   return {
     name: 'wxt:noop-background',
-    resolveId(id) {
-      if (id === virtualModuleId) return resolvedVirtualModuleId;
+    resolveId: {
+      filter: {
+        id: new RegExp(`^${virtualModuleId}$`),
```

Comment:
Same here. Does this work? No need to use regex if it's not needed.


```suggestion
        id: virtualModuleId,
```